app/views/api_users/show.html.haml

```diff
@@ -8,18 +8,14 @@
       %code.code.code-large
         = @api_key
 
-      %p.font-xsmall Your API key has also been emailed to you. Your key can be used for all registers.
+      %p Your API key has also been emailed to you. Your key can be used for all registers.
+
+      %h3.heading-medium Get this register as JSON
+      %p
+        %code.code.code-curl
+          curl https://#{session[:register].present? ? session[:register] : 'register-name'}.register.gov.uk/records.json?page-size=5000 --header "Authorization: #{@api_key}"
```

Comment:
This will not work if they register from an alpha register as the URL will be wrong.

Reply:
the URL is stored in the model as `register.url`

Reply:
Thanks Gids, i'm refactoring this now 👍